src/common/cmd.ts

```diff
@@ -0,0 +1,67 @@
+import { spawn } from 'node:child_process';
+
+let spawned = false;
+let stdErrStr = '';
+
+export async function runCommand(
```

Comment:
shouldn't it be in mc-utils for future reuse?

Reply:
yes, but I prefer as single task on future after some time of using and see evaluation